frontends/web/src/api/backend.ts

```diff
@@ -32,6 +32,8 @@ export interface ISuccess {
     errorCode?: string;
 }
 
+export type BackendErrorCode = 'userAbort' | 'addressNotFound';
```

Comment:
i wonder if it's better to have one enum of all backend error codes or scope them to the specific handlers. i think i prefer the latter a bit, as then it's very clear which function needs to handle which errors.